src/main/java/no/nav/veilarbvedtaksstotte/config/HealthConfig.java

```diff
@@ -59,14 +60,12 @@ public SelfTestChecks selfTestChecks(VeilarbarenaClient arenaClient,
                 new SelfTestCheck("SafClient", false, safClient),
                 new SelfTestCheck("VeilederOgEnhetClient", false, veiledereOgEnhetClient),
                 new SelfTestCheck("UnleashClient", false, unleashClient),
-                new SelfTestCheck("Ping database", true, () -> checkDbHealth(dataSourceHealthIndicator))
+                new SelfTestCheck("Ping database", true, () -> checkDbHealth(dataSourceHealthIndicator)),
+                new SelfTestCheck("DokarkivClient", false, dokarkivClient),
+                new SelfTestCheck("DokdistribusjonClient", false, dokdistribusjonClient),
+                new SelfTestCheck("RegoppslagClient", false, regoppslagClient)
```

Comment:
Er det  verdt å endre denne til critical = true?
For det vil vel ikke være mulig å generere PDF uten en adresse fra RegoppslagClient?

Reply:
Ja, det er et godt poeng. Men da tror jeg alle må bli critical = true. Slik det er nå er det kun databasen som er critical. Tanken bak det er at vedtaksstøtte vil kunne fungere delvis likevel, f.eks. kan man fint skrive utkast og lagre endringer, selv om man midlertidig ikke kan sende brev.